docs/tutorial/multi_parameters.md

```diff
@@ -9,3 +9,5 @@ When multi parameters are enabled, a different set of parameters are selected fo
 - Larger memory usage during execution.
 
 To disable it, you can use `parameter_selection_strategy=fhe.ParameterSelectionStrategy.MONO` configuration option.
+
+When enabled, you can choose a strategy on how the multiple parameters are added and assigned in the circuit, see **multi\_parameter\_strategy** in [configuration](../howto/configure.md#options).
```

Comment:
Why `are added and assigned` and not `are assigned`?

Reply:
because it introduces new partitions.
"a strategy that defines the circuit partitionning and how multi-parameters are assigned" ?

Reply:
Updated to
Set the level of circuit partionning when using `fhe.ParameterSelectionStrategy.MULTI`.